.gitignore

```diff
@@ -47,3 +47,6 @@ tool_test_output*
 docs/_build
 
 .venv
+
+# ctags file
+tags
```

Comment:
What is this file ? Is that common enough to warrant inclusion here ?

Reply:
It shouldn't be here, sorry! It's a vim/emacs thing - a tag index generated by ctags (code navigation without a 'guess-the-button' IDE). I'll remove it.